src/corva/handlers.py

```diff
@@ -544,3 +562,55 @@ def _get_custom_event_type_by_raw_aws_event(
         if events:
             return event_type, handler
     return None, None
+
+
+def _merge_events(aws_event: Any, data_transformation_type: Type[RawBaseEvent]) -> Any:
+    """
+    Merges incoming aws_events into one.
+    Merge happens differently, depending on app type.
+    Only "scheduled" and "stream" type of apps can be processed here.
+    If somehow any other type is passed - raise an exception
+    """
+    if data_transformation_type == RawScheduledEvent:
+        # scheduled event
+        if not isinstance(aws_event[0], dict):
+            aws_event = list(itertools.chain(*aws_event))
+        is_depth = aws_event[0]["scheduler_type"] == SchedulerType.data_depth_milestone
```

Comment:
can you compare str and enum using `==`? or should it be:
```python
aws_event[0]["scheduler_type"] == SchedulerType.data_depth_milestone.value
```

Reply:
`_merge_events` function receives aws event as dict, because parsing is done after it https://github.com/corva-ai/python-sdk/blob/DEVC-496-be-stream-spike-contact-devcenter-team-to-update-python-sdk-to-increase-data-batch-size/src/corva/handlers.py#L93